doc/article/en-US/angular1-to-aurelia.md

```diff
@@ -0,0 +1,526 @@
+		
+		  get fullName() {
+		    return `${this.firstName} ${this.lastName}`;
+		  }
+		
+		  submit() {
+		    this.previousValue = this.fullName;
+		    alert(`Welcome, ${this.fullName}!`);
+		  }
+		}
+	</source-code>
+</code-listing>
+
+> Note: Controller Migration
+> Later versions of Angular supported the "controller as" syntax, allowing developers to declare viewmodel properties on the `this` property. However, `$scope` was still necessary for watches and signalling.) Your application will be easier to migrate to Aurelia if you are only using `$scope` for signalling.
+
+### Migrating your Views
+
+Fourth step, part two - we'll migrate the views. Aurelia views and Angular views are very similar. However, Aurelia views follow the Web Components specification, requiring views to be wrapped in a `<template>` tag. Further, references to custom views or custom elements must be included in the view using the `<require>` tag.
+
+#### Naming Conventions
+
+In Angular, a view was often strongly coupled to a viewmodel (or controller) instance, using something similar to the following code:
+
+<code-listing heading="Angular View">
+	<source-code lang="ES 2015/2016">		<section data-ng-controller="myController as MyCtrl">
+		...
+		</section>
+	</source-code>
+</code-listing>
+
+Aurelia, by default, prefers convention over configuration. Aurelia's default convention is that a viewmodel will have a view with the same name. To take advantage of this, keep a view in the same directory alongside its viewmodel, and make sure that the HTML file and Javascript file share the same filename.
+
+> Note: Convention vs configuration
+> This behavior is also configurable using the ViewLocator service. By configuring Aurelia's ViewLocator, you can override Aurelia's defaults. Views and viewmodels can be linked by location, by a file naming convention, or by other means.
+
+#### Changing Angular interpolation to Template String Interpolation
+
+Angular, by default, uses a double-bracket syntax for interpolation, like so:
+
+<code-listing heading="Angular Interpolation">
+	<source-code lang="HTML">
+		<div id="hello">Hello, my name is {{name}}!</div>
+	</source-code>
+</code-listing>
+
+Aurelia makes use of ES6 template strings in its templates, so the Aurelia version becomes this:
+
+<code-listing heading="Aurelia Interpolation">
+	<source-code lang="HTML">
+		<div id="hello">Hello, my name is ${name}!</div>
+	</source-code>
+</code-listing>
+
+Aurelia's template strings use pipe syntax for ValueConverters (filters in Angular).
+
+#### Template behaviors
+
+`ng-show` becomes `show.bind`. `ng-if` becomes `if.bind`. `ng-click` becomes `click.delegate`.
+
+Angular's repeat syntax looks like this:
+
+<code-listing heading="Angular Interpolation">
+	<source-code lang="HTML">
+		<ul data-ng-repeat="item in items">
+			<li id="{{item.id}}">{{item.name}}</li>
+		</ul>
+	</source-code>
+</code-listing>
+
+In Aurelia, the repeat is not on the parent element. It looks like this:
```

Comment:
@EisenbergEffect, i think it would be worth explaining why different approach was chosen - I guess Your comments would be very much appreciated here to avoid frustrations :)